main.go

```diff
@@ -1,4 +1,17 @@
+/*
+ * Craftgate API Client
+ *
+ * Contact: destek@craftgate.io
+ */
+
 package main
 
+import (
+	"fmt"
+	craftgate "github.com/craftgate/craftgate-go-client/v1.0.0/adapter"
+)
+
 func main() {
```

Comment:
bu dosya direk uçacaktı sanırım abi bizim kendi mainimizin olmasına gerek yok test için kullanıyorduk, ama hiç mail olmazsa da proje ötüyor ben boş main bırakmıştım o yüzden

Reply:
evet abi ici bos aslında zaten, diger client orneklerinde de bu sekilde bir kullanım var. ci 'da patlıyor go dosyası hic olmayınca. ici bos bir main gibi düsünebiliriz